MiguelAValdesG/Hackerrank/JavaHashTablesRansomNote.java

```diff
@@ -0,0 +1,74 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+
+class Result {
+
+  public static boolean checkMagazine(List<String> magazine, List<String> note) {
+    HashMap<String, Integer> wordsInMagazine = new HashMap<>();
```

Comment:
Always using interface type when declaring a collection
https://www.codejava.net/java-core/collections/18-java-collections-and-generics-best-practices